extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -356,6 +384,11 @@ private Map<String, String> getAttributeTokens(ConnectedLDAPConfiguration config
                 tokens.put(TokenName.canonicalize(attr.getId(),
                         LDAP_ATTRIBUTE_TOKEN_PREFIX), attr.getString());
             }
+            String domainName = getDomainToken(credentials);
+            if (domainName != null) {
+                String tokenName = TokenName.canonicalize(LDAP_DOMAIN_TOKEN, LDAP_ATTRIBUTE_TOKEN_PREFIX);
+                tokens.put(tokenName, domainName);
+            }
```

Comment:
This shouldn't be within the `try`/`catch` block that relates to LDAP errors that may be encountered while retrieving attributes, as it's entirely independent of all that. For readability, it should also be visibly separated from the block that precedes it, which is already documented as having a different purpose.

I'd recommend just moving this outside the `try`/`catch`, making sure to add:

* A blank line to visibly separate the block from everything else
* A high-level comment covering the intent of the block.